docs/additional-functionality/rapids-udfs.md

```diff
@@ -320,4 +320,4 @@ The following configuration settings are also relevant for GPU scheduling for Pa
     of them are waiting for their second semaphore.
 
 To find details on the above Python configuration settings, please see the [RAPIDS Accelerator for
-Apache Spark Configuration Guide](../configs.md). Search 'pandas' for a quick navigation jump.
+Apache Spark Configuration Guide](https://nvidia.github.io/spark-rapids/docs/configs.html). Search 'pandas' for a quick navigation jump.
```

Comment:
This link should be kept relative so those browsing in Github pages stay on pages, and those browsing the markdown in the Github repo stay in the repo.

Reply:
Aha, the reason why i changed it is due to ../configs.md is not working properly on the gh-pages.
Now I figured out why.
Just fixed the `../configs.md`. Please take a look again.